docker-entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
can you have this script print out how it's supposed to be invoked if one of the required ENV is not set? (looks like only $BUCKETNAME), or if one of the required capabilities is not passed